FirebaseDataConnect/Tests/Integration/IntegrationTests.swift

```diff
@@ -12,10 +12,218 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import Foundation
+import XCTest
 
+import FirebaseCore
 @testable import FirebaseDataConnect
 
-import XCTest
+final class IntegrationTests: XCTestCase {
+  class func setupFirebaseApp() {
+    if FirebaseApp.app() == nil {
+      let options = FirebaseOptions(googleAppID: "0:0000000000000:ios:0000000000000000",
+                                    gcmSenderID: "00000000000000000-00000000000-000000000")
+      options.projectID = "fdc-test"
+      FirebaseApp.configure(options: options)
+    }
+  }
+
+  private let setupQ = DispatchQueue(
+    label: "setupFDCIntegrationsTests",
+    autoreleaseFrequency: .workItem
+  )
+  private var _setupComplete = false
+  private var setupComplete: Bool {
+    set { setupQ.sync { _setupComplete = newValue } }
+    get { setupQ.sync { _setupComplete }}
+  }
+
+  override class func setUp() {
+    setupFirebaseApp()
+    DataConnect.kitchenSinkClient.useEmulator(port: 3628)
+  }
+
+  override func setUp(completion: @escaping ((any Error)?) -> Void) {
+    Task {
+      print("setupComplete \(self.setupComplete) \(self)")
```

Comment:
Do you want to keep these "print" statements in the code? It looks like they were useful during your development, but may not make sense to keep them in there.

Reply:
Good catch. Updated and removed these.